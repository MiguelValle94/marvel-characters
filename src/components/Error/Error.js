import React from 'react';
import PropTypes from 'prop-types';

export default function Error({ message }) {
  return (
    <div className='alert alert-danger' role='alert'>
      {message}
    </div>
  );
}

Error.propTypes = {
  message: PropTypes.string.isRequired
};
