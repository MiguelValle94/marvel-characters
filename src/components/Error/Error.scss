@import '../../styles/variables.scss';

.error {
  padding: var(--space);
}
