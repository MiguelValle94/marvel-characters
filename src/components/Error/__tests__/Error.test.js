import React from 'react';
import { shallow } from 'enzyme';

import Error from '../Error';

const setUpShallow = (message) => {
  const component = shallow(<Error message={message} />);
  return component;
};

describe('Error', () => {
  it('should match snapshot', () => {
    const component = setUpShallow('Some error message');
    expect(component).toMatchSnapshot();
  });
});
