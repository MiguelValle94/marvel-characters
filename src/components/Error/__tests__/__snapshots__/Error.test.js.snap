// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match snapshot 1`] = `
<div
  className="alert alert-danger"
  role="alert"
>
  Some error message
</div>
`;
